config/rails.yml

```diff
@@ -77,9 +77,12 @@ Layout/EmptyLinesAroundMethodBody:
 Layout/EmptyLinesAroundModuleBody:
   Enabled: true
 
-Layout/FirstParameterIndentation:
+Layout/IndentFirstArgument:
```

Comment:
I think it is preferable to move in ASCII order.

Reply:
Done!